commits.go

```diff
@@ -199,9 +199,13 @@ type CreateCommitOptions struct {
 	Branch        *string         `url:"branch" json:"branch"`
 	CommitMessage *string         `url:"commit_message" json:"commit_message"`
 	StartBranch   *string         `url:"start_branch,omitempty" json:"start_branch,omitempty"`
+	StartSha      *string         `url:"start_sha,omitempty" json:"start_sha,omitempty"`
```

Comment:
To keep things consistent and inline with the Go naming guidelines:

```suggestion
	StartSHA      *string         `url:"start_sha,omitempty" json:"start_sha,omitempty"`
```